<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LHelpWidget</class>
 <widget class="QWidget" name="LHelpWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>978</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lobster Help</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="contentsTab">
       <attribute name="title">
        <string>contents</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QListWidget" name="listWidget">
          <item>
           <property name="text">
            <string>index</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>script</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>plugins</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>mainwindow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>oscilloscope</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>watchwindow</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>logsystem</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="homeButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="backwardButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="forwardButton">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTextBrowser" name="textBrowser"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
